src/zope/configuration/fields.py

```diff
@@ -38,21 +40,33 @@
 
 @implementer(IFromUnicode)
 class PythonIdentifier(TextLine):
-    """This field describes a python identifier, i.e. a variable name.
     """
-    def fromUnicode(self, u):
-        return u.strip()
+    This field describes a python identifier, i.e. a variable name.
```

Comment:
Hmm, I didn't know [PEP 257 permitted the summary line under the triple-quotes](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings).

Reply:
It does 😄 :

> The summary line may be used by automatic indexing tools; it is important that it fits on one line and is separated from the rest of the docstring by a blank line. The summary line may be on the same line as the opening quotes or on the next line. 

There's an example:

> The docstring in this example contains two newline characters and is therefore 3 lines long. The first and last lines are blank:

> def foo():
>    """
>    This is the second line of the docstring.
>    """
> To illustrate:
>
> >>> print repr(foo.__doc__)
> '\n    This is the second line of the docstring.\n    '
> >>> foo.__doc__.splitlines()
> ['', '    This is the second line of the docstring.', '    ']
> >>> trim(foo.__doc__)
> 'This is the second line of the docstring.'

> Once trimmed, these docstrings are equivalent:
> 
> def foo():
>     """A multi-line
>    docstring.
>    """
>
> def bar():
>    """
>    A multi-line
>  docstring.
>    """

At least in Emacs, this is known as "django" style docstrings.

Reply:
Yup, my comment was just a TIL.